src/bci_build/package/__init__.py

```diff
@@ -1479,7 +1484,7 @@ def generate_disk_size_constraints(size_gb: int) -> str:
 
 SORTED_CONTAINER_IMAGE_NAMES = sorted(
     ALL_CONTAINER_IMAGE_NAMES,
-    key=lambda bci: f"{ALL_CONTAINER_IMAGE_NAMES[bci].os_version}-{ALL_CONTAINER_IMAGE_NAMES[bci].name}",
+    key=lambda bci: f"{ALL_CONTAINER_IMAGE_NAMES[bci].os_version}-{ALL_CONTAINER_IMAGE_NAMES[bci].uid}",
```

Comment:
that change seems unrelated?

Reply:
It is required, because `ObsPackageBase` doesn't have a `name` property